src/cf-typography/usage.md

```diff
@@ -7,30 +7,25 @@ and has more basic typography patterns.
 ## Contents
 
 - [Variables](#variables)
-- [Pull quote](#pull-quote)
-- [Micro copy](#micro-copy)
-- [Short description](#short-description)
-- [Date](#date)
+  - [Color variables](#color-variables)
+- [Running text](#running-text)
```

Comment:
Is the "Running text" sectioning really useful here? If so, is it used correctly? The date atom doesn't really go in running text, and links with icons do go in running text, to call out two obvious inconsistencies.

I think I'd suggest just ditching "Running text" as a grouping.